### DISCUSSION TITLE
RSS XML formatting incorrect

### DISCUSSION THREAD

**Discussion by Obspogon** [11ty/eleventy#2377] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/72287594/167704274-2ee31bc5-78d2-4883-aec3-b745ef54fae9.png)

![image](https://user-images.githubusercontent.com/72287594/167704711-c5bc755e-e1eb-4c44-b831-0c1cd113c584.png)

Above image from building feed from Nunjucks template, then Firefox error when accessing feed directly. Fixed when I retype `xml`
Using [example from docs](https://www.11ty.dev/docs/plugins/rss/)

**Comment by cfjedimaster** [comment, marked as answer]
When you view source, do you have white space before/above the XML tag? If so you need to remove that.
